examples/IMEX/diffusion-nl/diffusion-nl.cxx

```diff
@@ -21,7 +21,7 @@
 
 class DiffusionNL : public PhysicsModel {
 protected:
-  int init(bool restarting) {
+  int init(bool) {
     // Get the input parameter alpha
     auto opt = Options::root();
```

Comment:
`auto&`? Might not matter here, I guess

Reply:
Ah yes, good catch! Good to get into that habit, so that docs etc. get saved.